### DISCUSSION TITLE
Asset Permissions

### DISCUSSION THREAD

**Discussion by FlorinBuffet** [requarks/wiki#6632] [Error / Bug Report] [answered]
Hi there
I have some articles public with the rule "Tag = public". Unfortunately this then does not apply to the images, probably as they don't have a tag associated.
Similar Issue (but with paths and not tags): #5663  
Is this known like that or can this be fixed somehow?
Best Regards
Florin

**Comment by NGPixel** [comment, marked as answer]
Assets only works with path-based rules.

**Reply by FarmDoc** [reply]
Thanks for the clarification.

Would it then help to just create a second rule "**read:assets** - **path starts with** - **/** ", so that they see all assets?
(For me it does not seem to work this way).
